pkgs/development/python-modules/skidl/default.nix

```diff
@@ -1,45 +1,37 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, requests
 , future
 , kinparse
-, enum34
 , pyspice
 , graphviz
-, pillow
-, cffi
 }:
 
 buildPythonPackage rec {
   pname = "skidl";
-  version = "unstable-2020-09-15";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "xesscorp";
     repo = "skidl";
-    rev = "551bdb92a50c0894b0802c0a89b4cb62a5b4038f";
-    sha256 = "1g65cyxpkqshgsggav2q3f76rbj5pzh7sacyhmhzvfz4zfarkcxk";
+    rev = version;
+    sha256 = "1m0hllvmr5nkl4zy8yyzfgw9zmbrrzd5pw87ahd2mq68fjpcaqq5";
   };
 
   propagatedBuildInputs = [
-    requests
     future
     kinparse
-    enum34
     pyspice
```

Comment:
Not sure about this one https://github.com/xesscorp/skidl/blob/1.0.0/setup.py#L33

Reply:
It's used in e.g. https://github.com/xesscorp/skidl/blob/7c7e07e4d48f8f629dd6e66c94035de86bae67fe/skidl/tools/spice.py so I left it in.